core/tests/ts-integration/tests/api/web3.test.ts

```diff
@@ -277,55 +277,55 @@ describe('web3 API compatibility tests', () => {
         expect(eip1559ApiReceipt.maxPriorityFeePerGas).toEqual(0n);
     });
 
-    test('Should test getFilterChanges for pending transactions', async () => {
```

Comment:
You can use `test.skip` (i.e., add `.skip`) to temporarily disable a test.

Reply:
honestly, I prefer it as it is, it's super easy to miss test.skip and get confused while this is much more visible